feature-libs/product/configurators/rulebased/cpq/cpq-configurator-rest.service.ts

```diff
@@ -72,4 +101,13 @@ export class CpqConfiguratorRestService {
       `${CPQ_CONFIGURATOR_VIRTUAL_ENDPOINT}/api/configuration/v1/configurations/${configId}/display`
     );
   }
+
+  protected callUpdateAttribute(
+    updateAttribute: Cpq.UpdateAttribute
+  ): Observable<Cpq.ConfigurationCreatedResponseData> {
```

Comment:
why does call**Update**Attribute return on observable of  Configuration**Created**ResponseData?

Reply:
fixed